YouVsVirus/Assets/Scripts/Infection/ProbabilityBasedInfection.cs

```diff
@@ -34,10 +34,7 @@ public override void Update()
         {
             if (infectionControl.IsNewDay())
             {
-                // nobody becomes infectious or dies at the party
-                // due to the much shorter time scale
-                if(LevelSettings.GetActiveSceneName() != "YouVsVirus_Leveldisco")
-                    UpdateCondition();
+                UpdateCondition();
```

Comment:
Can you explain why you changed this?
Does not seem to have to da anything with the Q-key or the demo level...

Reply:
Should have done it separately. Was superfluous since we handle this in the level settings now.